pkg/cmd/cli/cmd/set/set.go

```diff
@@ -57,3 +59,29 @@ func NewCmdSet(fullName string, f *clientcmd.Factory, in io.Reader, out, errout
 	templates.ActsAsRootCommand(set, []string{"options"}, groups...)
 	return set
 }
+
+const (
+	setImageLong = `
+Update existing container image(s) of resources.
+`
+
+	setImageExample = `	# Set a deployment configs's nginx container image to 'nginx:1.9.1', and its busybox container image to 'busybox'.
```

Comment:
This seems a tab, use 2 spaces in examples. BTW we upstreamed our templating system with some improvements including correcting examples tabbing automatically, adding a `post-rebase` issue to myself to reuse templating from Kube. Once I do that we'll no longer need to worry about this.